docs/Documentation/Features/Menus/Menu.md

```diff
@@ -77,13 +77,28 @@ The three basic optional settings.
 By default, the name and description of the quest item is displayed when hovering over the item.
 You can overwrite this by using the `text` setting.
 Both [color codes](https://minecraft.gamepedia.com/Formatting_codes) and [variables](../../Scripting/Building-Blocks/Variables-List.md) are supported.
+The text can be provided as a single string with newlines, a multi-line string, or a list of strings, see examples.
 
-``` YAML title="Example"
+``` YAML title="List Example"
+skeletonQuestDone:
+  item: "questDoneItem"
+  text:
+    - "&2Quest reputation: &6&l%point.quest_reputation.amount%" #(1)!
+    - "Complete quests to gain reputation!" #(2)!
+```
+
+``` YAML title="String with Newlines Example"
+skeletonQuestDone:
+  item: "questDoneItem"
+  text: "&2Quest reputation: &6&l%point.quest_reputation.amount%\nComplete quests to gain reputation!" #(1)! #(2)!
+```
+
+``` YAML title="Multi-line String Example"
 skeletonQuestDone:
   item: "questDoneItem"
-   text:
-     - "&2Quest reputation: &6&l%point.quest_reputation.amount%" #(1)!
-     - "Complete quests to gain reputation!" #(2)!
+  text: |-
+    &2Quest reputation: &6&l%point.quest_reputation.amount% #(1)!
+    Complete quests to gain reputation! #(2)!
```

Comment:
@SaltyAimbOtter What do we need to do here to get the #(1) and #(2) working again for all code blocks?

Reply:
We are lucky, there seems to be a new feature that allows annotations not only in comments but also in other syntax elements like this multi-line string.
https://squidfunk.github.io/mkdocs-material/reference/code-blocks/?h=code#custom-selectors
I could not get it to work though. Perhaps you need to have a look, maybe I'm misunderstanding something. If not we need to file a bug @mkdocs-material.

Reply:
In case this was not clear: I will take care of this and submit a PR to your branch / commit directly. Then I will merge this PR. Might just take a little while 😁